### DISCUSSION TITLE
Can I register multiple preserved keys for one function?

### DISCUSSION THREAD

**Discussion by Rohupt** [saschanaz/ime-rs#12] [General]
As the title says. I want to register several options to toggle the IME on/off, but as far as I see simply adding elements to the `keys` array in `preserved_keys.rs` won't work.

Is it possible in the first place?

**Comment by saschanaz** [comment]
I think it should work... Did you also give a new GUID for key_guid?

**Reply by Rohupt** [reply]
Oh yeah, I didn't. I will try it now.

**Reply by Rohupt** [reply]
It works! But still, the one I want (Left Shift + Right Shift) is not working. I'm adding two combinations (key = LSHIFT, mod = RSHIFT and vice versa). Let's see if I can make it happen.
